static/scss/partials/onboarding.scss

```diff
@@ -301,6 +301,19 @@
             @include text-title-2xs;
         }
     }
+
+    .c-premium-onboarding-domain-registration-form {
+        display: flex;
+        flex-direction: column;
+
+        @media #{$mq-md} {
+            display: inline;
+        }
+    }
+
+    .subdomain-banner-input {
+        margin-bottom: $spacing-md;
```

Comment:
This works fine, but keeping composable CSS in mind (see [article I linked earlier](https://non-traditional.dev/encapsulated-css-the-key-to-composable-layouts-94f11c177cc1?gi=1d779f6274ce)), it would be better to set `gap: $spacing-md` on the container, rather than a margin on an element that might theoretically be re-used elsewhere. (Although in this case the input _already_ gets a margin from `forms.scss` that you'd also have to override, so it's already not as composable as it could be.)

It doesn't matter for this PR (so no need to change it) since this element isn't really used elsewhere I think, but hopefully it's educational :)

Reply:
Ah, I've never heard of the gap property before. Will keep it in mind!